types/function_list_builder_test.go

```diff
@@ -43,6 +43,37 @@ func TestBuildSingleMatchingFunction(t *testing.T) {
 	}
 }
 
+func TestBuildMultiMatchingFunction(t *testing.T) {
+
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+
+		functions := []requests.Function{}
+		annotationMap := make(map[string]string)
+		annotationMap["topic"] = "topic1,topic2,topic3"
```

Comment:
I would like a test-case without the separator and with a string like this: `,`

Reply:
@rgee0 ^

Reply:
Theyre there now